monkey/infection_monkey/ransomware/ransomware_payload.py

```diff
@@ -1,27 +1,64 @@
 import logging
+from pathlib import Path
+
+from infection_monkey.ransomware.valid_file_extensions import VALID_FILE_EXTENSIONS_FOR_ENCRYPTION
+from infection_monkey.utils import bit_manipulators
+from infection_monkey.utils.dir_utils import (
+    file_extension_filter,
+    filter_files,
+    get_all_regular_files_in_directory,
+    is_not_shortcut_filter,
+    is_not_symlink_filter,
+)
+from infection_monkey.utils.environment import is_windows_os
 
 LOG = logging.getLogger(__name__)
 
+CHUNK_SIZE = 64
+EXTENSION = ".m0nk3y"
+
 
 class RansomewarePayload:
     def __init__(self, config: dict):
-        self.config = config
+        LOG.info(f"Windows dir configured for encryption is " f"{config['windows_dir']}")
+        LOG.info(f"Linux dir configured for encryption is " f"{config['linux_dir']}")
```

Comment:
These should be remove in this PR, because it's where they actually get used.

Reply:
I think logging this information is still valuable.